fetch.bs

```diff
@@ -2689,9 +2738,29 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
      <a for=request>response tainting</a> to
      "<code>opaque</code>".
 
-     <li><p>Return the result of performing a <a>scheme fetch</a>
-     using <var>request</var>.
+     <li><p>Let <var>noCorsResponse</var> be the result of performing a <a>scheme fetch</a> using
+     <var>request</var>.
      <!-- file URLs end up here as they are not same-origin typically. -->
+
+     <li>
+      <p>If <var>noCorsResponse</var> is not a <a>filtered response</a> and the <a>CORB check</a>
+      with <var>request</var> and <var>noCorsResponse</var> returns <b>blocked</b>, then:
+
+      <ol>
+       <li><p>Let <var>oldNoCorsResponse</var> be <var>noCorsResponse</var>.
+
+       <li>
+        <p>Set <var>noCorsResponse</var> to a new <a for=/>response</a> whose
```

Comment:
If you do the immediate return as suggested above, you also don't need to do the variable juggling here.

Instead, you can call this something more meaningful, like `sanitizedResponse`.

Reply:
Okay.